### PR TITLE
Add ipv6 to filters in Node Finder

### PR DESCRIPTION
**Pull request by zaelgohary**
### Description

- Add ipv6 to filters in node finder
- Update Grid Client filter to include hasIPv6 and include the new filter in Playground
- Update farm filters to include node_has_ipv6 in grid client, grid proxy & playground

[ipv6.webm](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/74c93741-94c7-44cc-b431-08442c460d40)

### Related Issues

- https://github.com/threefoldtech/tfgrid-sdk-ts/issues/155

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [x] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
there is a switch for ipv6 while deploying any solution, but this switch doesn't filter the nodes as it wasn't supported. please make sure it filter the nodes there too.

**Comment by zaelgohary** [comment]
> IPv6 filter doesn't work in micro VM 

I was using a wrong filter query. It should've be ipv6 only as seen below. Should be fixed now. 

image

**Review by MohamedElmdary — APPROVED** [review]

**Review by maayarosama — APPROVED** [review]

**Review by AhmedHanafy725 — CHANGES_REQUESTED** [review]
I have opened an issue for supporting this query on farms too to show only the farms that have ipv6 in case it was selected

File: packages/grid_client/src/primitives/nodes.ts [hunk 1]
```diff
@@ -390,7 +390,7 @@ class Nodes {
       free_hru: Math.ceil(this._g2b(options.hru)) || "",
       free_ips: options.publicIPs ? 1 : "",
       ipv4: options.accessNodeV4,
-      ipv6: options.accessNodeV6,
+      ipv6: options.hasIPv6,
```
[the remark this answers is not in the document]
Comment (zaelgohary) [reply packages/grid_client/src/primitives/nodes.ts:393]: Yes, you're right but this is how it's done in Grid Proxy as explained [here](https://github.com/threefoldtech/tfgrid-sdk-ts/pull/2738#issuecomment-2132234885). Should I open an issue in the Grid proxy to change the query param to has_ipv6?

[the remark this answers is not in the document]
Comment (zaelgohary) [reply packages/grid_client/src/primitives/nodes.ts]: The swagger wasn't updated. I added the filter to proxy and updated both grid client & playground.

**Review by maayarosama — APPROVED** [review]

**Review by AhmedHanafy725 — APPROVED** [review]

**Pull request merged**